MissedRevenueHeatMapGenerator.py

```diff
@@ -29,7 +29,7 @@ def calculate_run_rate(self, sku, branch, journal, period):
         #return run rate for a period
         pass
     
-    def calculate_missed_revenue(self, sku, branch, retail_price, qty_of_lost_sales):
+    def calculate_missed_revenue(self, sku, branch, retail_price, journal):
```

Comment:
I realized that I should calculate the missed revenue for a specific time period.

I have thought about the shape of the various inputs of each method, will show this in the comments